test/codegen/__snapshots__/codegen.js.snap

```diff
@@ -1196,6 +1208,10 @@ import com.fasterxml.jackson.annotation.*;
 public enum LaptopMake {
    Apple,
    Microsoft,
+   @Override
```

Comment:
I am not sure if enums in java store functions.

Reply:
Yes it does, because I have created one java program where I declared a enum and inside it defined a function and it worked totally final without any

Reply:
Okay, thanks for clarification. I'm not much proficient in Java. @mttrbrts can you please review this?

Reply:
```
public class Main{
    public enum State {
        MA,
        NY,
        CO,
    
        // Override the toString() method for the entire enum type
        @Override
        public String toString() {
            return name();
        }
    }
    public static void main(String[] args) {
        // Using the enum constants
        State state1 = State.MA;
        State state2 = State.NY;
        String ma = state1.toString();
        
        // Printing the enum constants
        System.out.println("State 1: " + ma); // This will call the overridden toString() method
    }
}

```

Reply:
Hey @sanketshevkar Have one question, How can I pass extra parameters like` parameters.filewriter` as show in the code.. I mean to say can we pass other parameter like this `parameters.x` ?
https://github.com/accordproject/concerto-codegen/blob/7c14d706387e371d22c36ccdb17fda08b4a2f1e7/lib/codegen/fromcto/java/javavisitor.js#L142-L146

Reply:
> yes, you can. it is an options object on the visitor. (answered by @dselman on the Tech WG call)

Okay ... I am assuming is that something like this `parameters.options`

Reply:
Hi! is there any update or any feedback on this pr? @dselman @mttrbrts